src/XsdToMarkdown/ConvertXsdToMarkdownCommand.cs

```diff
@@ -28,7 +49,7 @@ private string PageId(PageType type, string name = null)
         {
             return type switch
             {
-                PageType.SchemaRoot => $"{this.Xsd.SchemaName}".ToLowerInvariant(),
+                PageType.SchemaRoot => $"{this.Xsd.SchemaName}/index".ToLowerInvariant(),
```

Comment:
Can we just use `/` instead of adding `/index`?

Reply:
No, the `index` part of the id becomes `index.md`.